docs/import/add_disaster.js

```diff
@@ -325,3 +563,29 @@ function createOptionFrom(innerTextAndValue) {
       .val(innerTextAndValue)
       .prop('id', innerTextAndValue);
 }
+
+/**
+ * Utility function for getting current data. May return undefined if
+ * currentDisaster hasn't been set yet but currentDisaster gets set as soon
+ * as we get all the disasters from firestore.
+ * @return {Object}
+ */
+function getCurrentData() {
+  return disasterData.get(currentDisaster);
+}
+
+/**
+ * Utility function for getting current layers.
+ * @return {Array<Object>}
+ */
+function getCurrentLayers() {
+  return getCurrentData()['layers'];
+}
+
+/**
+ * Sets the current disaster so getCurrentData works for testing.
```

Comment:
I think this page should probably be disaster-setting aware. Ram's PR puts the current disaster name into localStorage. So you can use that in this PR and have Cypress set the value in localStorage.

Reply:
Discussed offline - will handle in a follow up PR. Added a TODO.